csharp/src/Apache.Arrow/Ipc/ArrowStreamWriter.cs

```diff
@@ -343,7 +343,7 @@ private protected void WriteRecordBatchInternal(RecordBatch recordBatch)
             FinishedWritingRecordBatch(bodyLength + bodyPaddingLength, metadataLength);
         }
 
-        private Tuple<ArrowRecordBatchFlatBufferBuilder, VectorOffset> PreparingWritingRecordBatch(RecordBatch recordBatch)
+        private protected Tuple<ArrowRecordBatchFlatBufferBuilder, VectorOffset> PreparingWritingRecordBatch(RecordBatch recordBatch)
```

Comment:
Is this change necessary? I don't see additional callers of this method.

Reply:
reverted back